extensions/objects/protractor.js

```diff
@@ -179,6 +179,19 @@ var ParameterNameEditor = function(elem) {
   };
 };
 
+var PositiveIntEditor = function(elem) {
+  return {
+    setValue: async function(value) {
+      await elem.element(by.tagName('input')).clear();
+      await elem.element(by.tagName('input')).sendKeys(value);
```

Comment:
Can you use `action` functions here? I know it's not common yet in this file, but doing so here will mean less for us to migrate later

Reply:
Done